CMakeLists.txt

```diff
@@ -130,6 +130,7 @@ add_subdirectory(libevmasm)
 add_subdirectory(libyul)
 add_subdirectory(libsolidity)
 add_subdirectory(libsolc)
+add_subdirectory(solstdlib)
```

Comment:
Renamed this from `stdlib` to `solstdlib` in a commit as there were cryptic C++ errors, and I wasn't sure if it was due to the name. Need to double check.